coordinator/src/main/scala/com/socrata/datacoordinator/service/DynamicPortMap.scala

```diff
@@ -0,0 +1,17 @@
+package com.socrata.datacoordinator.service
+
+import com.typesafe.scalalogging.slf4j.Logging
+
+trait DynamicPortMap extends Logging  {
```

Comment:
Don't make a trait that extends "Logging" (actually, just don't use the Logging trait at all); the logger it produces will be named after the dynamic type of the _instance_ (as it uses getClass).